src/auto/injector.js

```diff
@@ -359,6 +359,8 @@ function annotate(fn, strictDi, name) {
  * * {@link auto.$provide#service service(class)} - registers a **constructor function**, `class`
  *     that will be wrapped in a **service provider** object, whose `$get` property will instantiate
  *      a new object using the given constructor function.
+ * * {@link auto.$provide#decorator decorator(name, decorFn)} - registers a **decorator function**,
+ *      `decorFn`, with the {@link auto.$injector $injector}, `name` the name of the provider to decorate.
```

Comment:
, `name` the name of the provider to decorate. -> ; `name` is the name of the provider to decorate.